### PR TITLE
💄 Use Semantic for Token View

### LINKED ISSUE #306: Use Semantic for Token View
Replace current token management view with Semantic.

## Current
Tokens

### PR DESCRIPTION
**Pull request by dankolbman**
Replaces the New Token view with Semantic components and refactors apollo components to use composed HOCs instead.

NewTokens

Closes #306

### CODE REVIEW

File: src/components/Tokens/TokenList.js [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
-          </span>
-        </div>
-      </li>
+          </Header>
+        </List.Content>
+        {node.node.creator && <Image avatar src={node.node.creator.picture} />}
+        <List.Content>
+          <List.Header>{node.node.name}</List.Header>
+          {node.node.creator && <>Created by {node.node.creator.username} </>}
+          <TimeAgo date={node.node.createdAt} />
```
Comment (XuTheBunny) [inline src/components/Tokens/TokenList.js]: add `live={false}` to `<TimeAgo />`

File: src/components/Tokens/TokenList.js [hunk 2]
```diff
@@ -1,30 +1,31 @@
 import React from 'react';
-import {Icon} from 'kf-uikit';
+import {Button, Header, Image, List, Icon} from 'semantic-ui-react';
+import TimeAgo from 'react-timeago';
 import CopyButton from '../CopyButton/CopyButton';
 
 const TokenList = ({tokens, deleteToken}) => (
-  <ul className="min-w-full list-reset">
+  <List>
```
Comment (XuTheBunny) [inline src/components/Tokens/TokenList.js]: `<List relaxed>` to increase the padding between list items

File: src/views/TokensListView.js [hunk 3]
```diff
@@ -1,86 +1,167 @@
...
-                  <TokenList
-                    tokens={data.allDevTokens.edges}
-                    deleteToken={name => onDelete(deleteToken, name)}
-                  />
-                );
-              }}
-            </Mutation>
+    <>
+      <Container>
+        <Header as="h3">Manage Developer Download Tokens</Header>
```
Comment (XuTheBunny) [inline src/views/TokensListView.js]: Maybe add top margins here, and align the header with the rest of content

Comment (dankolbman) [reply src/views/TokensListView.js]: This should be cleared up once the new header is in place.

File: src/forms/NewTokenForm.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  Button,
+  Form,
+  Message,
+  Segment,
+} from 'semantic-ui-react';
+
+const NewTokenForm = ({onSubmit, error, loading}) => (
+  <Segment>
+    <Form onSubmit={onSubmit} className="FileEditor">
+      <Form.Field>
```
Comment (XuTheBunny) [inline src/forms/NewTokenForm.js]: Maybe use the `Input` component here, and mark it as required.
```
<Form.Field required>
        <label>Name:</label>
        <Input
          type="text"
          name="name"
          className="FileEditor--Input"
          placeholder="Token Name"
        />
      </Form.Field>
```

**Pull request merged**